docs/configuration/cluster_manager/cluster_ssl.rst

```diff
@@ -43,7 +43,9 @@ verify_certificate_hash
 
 verify_subject_alt_name
   *(optional, array)* An optional list of subject alt names. If specified, Envoy will verify
-  that the server certificate's subject alt name matches one of the specified values.
+  that the server certificate's subject alt name matches one of the specified values. The names
+  support wildcard at the end. For example, ``foo.com/*`` will match certificate with subject alt
```

Comment:
Could you make it clear that this wildcard support is only for URI type SAN. And also for DNS type SAN, there's a wildcard support for "*.foo.com" style?